.pre-commit-config.yaml

```diff
@@ -1,20 +1,30 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
-    hooks:
-    -   id: check-yaml
-        files: '\.(yml|yaml)$'
-    -   id: end-of-file-fixer
-        files: '\.(yml|yaml)$'
-    -   id: trailing-whitespace
-        files: '\.(yml|yaml)$'
--   repo: https://github.com/ansible/ansible-lint
-    rev: v6.13.1
-    hooks:
-    -   id: ansible-lint
-        args: ["-c", ".ansible-lint.yml"]
-        files: '\.(yml|yaml)$'
-        language_version: python3
-        exclude: ansible/playbooks/prepare-data-dir.yml
-#       have to exclude these files as they're literally just empty files with one jinja2 inside so the yamllinter goes nuts
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: check-yaml
+    files: '\.(yml|yaml)$'
+  - id: end-of-file-fixer
+    files: '\.(yml|yaml)$'
+  - id: trailing-whitespace
+    files: '\.(yml|yaml)$'
+- repo: https://github.com/ansible/ansible-lint
+  rev: v6.13.1
+  hooks:
+  - id: ansible-lint
+    args: ["-c", ".ansible-lint.yml"]
+    files: '\.(yml|yaml)$'
+    language_version: python3
+    exclude: ansible/playbooks/prepare-data-dir.yml
+- repo: https://github.com/antonbabenko/pre-commit-terraform
+  rev: v1.50.0
+  hooks:
+      - id: terraform_fmt
+        exclude: ibm|azure|gcp
```

Comment:
Are we eventually bringing azure and gcp into scope?

Reply:
Yes, but not as part of this project. See internal [project overview ](https://docs.google.com/document/d/1-NIOmESzyx2X7csWB3Wz2DCtRqtNYd7jl5cqDJTGwNQ/edit)